frida-gum/Cargo.toml

```diff
@@ -18,6 +18,9 @@ num = "0.3.1"
 num-derive = "0.3.3"
 num-traits = "0.2.14"
 paste = "1"
+capstone = "0.8.0"
+capstone-sys = "0.12.0"
+libc = "0.2"
```

Comment:
If I am not mistaken, I think this was left over. `libc` is not used anywhere here.

Reply:
Fixed.